Documentation/data-collection.md

```diff
@@ -145,7 +145,14 @@ data:
     # version, phase, reason labels. When a csv is updated, the previous time
     # series associated with the csv will be deleted.
     - '{__name__="csv_abnormal"}'
-    # OCS metrics to be collected:
+    # (OCS-storage-management-team) Generic Storage metrics:
```

Comment:
oh please add a github alias.

Reply:
@s-urbaniak There is no particular alias for our team, we contribute to multiple ones. I have added @openshift/ocs-operator, it is part of OCP.